### ISSUE TITLE
NextJS - Type error when calling init functions

### ISSUE THREAD

**Issue by cehenriques** [supertokens/supertokens-node#230]
Hi.
I'm following the NextJS docs (https://supertokens.io/docs/emailpassword/nextjs/init), and on step 6 - "Call the init functions" we need to add the following function to the _app.ts file:

```ts
async function initNode() {
  const supertokensNode = await import('supertokens-node')
  const { backendConfig } = await import('../config/backendConfig')
  supertokensNode.init(backendConfig())
}
```

But I'm getting the following error on `supertokensNode.init(backendConfig())`:

```
Argument of type '{ framework: string; supertokens: { connectionURI: string; apiKey: string; }; appInfo: { appName: string; apiDomain: string; websiteDomain: string; apiBasePath: string; }; recipeList: RecipeListFunction[]; isInServerlessEnv: boolean; }' is not assignable to parameter of type 'TypeInput'.
  Types of property 'framework' are incompatible.
    Type 'string' is not assignable to type 'TypeFramework | undefined'.ts(2345)
```

Here's the backendConfig file that was auto-generated by the docs:

```ts
import EmailPasswordNode from "supertokens-node/recipe/emailpassword";
import SessionNode from "supertokens-node/recipe/session";
import { appInfo } from "./appInfo";

export const backendConfig = () => {
  return {
    framework: "express",
    supertokens: {
      // These are the connection details of the app you created on supertokens.io
      connectionURI:
        "<removed>",
      apiKey: "<removed>",
    },
    appInfo,
    recipeList: [EmailPasswordNode.init(), SessionNode.init()],
    isInServerlessEnv: true,
  };
};
```

I'm using supertokens-node@8.2.1

Thanks.

**Comment by rishabhpoddar** [comment]
Thanks for pointing this out. We will expose the `TypeInput` from the recipes using which you can do the following to fix this issue:
```
export const backendConfig = (): EmailPasswordNode.TypeInput => {
  return {
    framework: "express",
    supertokens: {
      // These are the connection details of the app you created on supertokens.io
      connectionURI:
        "<removed>",
      apiKey: "<removed>",
    },
    appInfo,
    recipeList: [EmailPasswordNode.init(), SessionNode.init()],
    isInServerlessEnv: true,
  };
};
```

That should fix this issue. Until then, you can make `backendConfig` return `any`.

**Comment by cehenriques** [comment]
> 

Thanks!

**Comment by rishabhpoddar** [comment]
Hi @cehenriques 

We have released a new version of the node SDK (v8.3.1) which exposes all the types. So you can do:
```
import { TypeInput } from "supertokens-node/types";

export const backendConfig = (): TypeInput => {
```

And the error should go away!
